ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos-intended-config.j2

```diff
@@ -335,6 +335,8 @@
 {% include 'eos/platform-apply.j2' %}
 {# EOS CLI #}
 {% include 'eos/eos-cli.j2' %}
+{# influxdb #}
```

Comment:
The `eos-cli.j2` should always come last (just before the custom templates).
Please identify where on EOS InfluxDB is configured and insert the template correctly.

Reply:
Added the template below cvx and above ip-security. Tested on lab.